logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserModel.kt

```diff
@@ -86,6 +86,13 @@ data class SelfUser(
     override val availabilityStatus: UserAvailabilityStatus
 ) : User()
 
+data class OtherUserMinimized(
```

Comment:
Only one question, what's `minimized` stands for ?  I don't know if it's related to minimized in the calling view. But maybe we can improve this name for something like `OtherUserPreview` ?

Not a blocker, at all, just maybe to be changed later 💪

Reply:
nothing related to calling, just light version of `OtherUser`

Reply:
nothing related to calling, just light version of `OtherUser`